src/components/paragraph.tsx

```diff
@@ -1,24 +1,35 @@
 // ----- Imports ----- //
 
 import React, { FC, ReactNode } from 'react';
-import { css } from '@emotion/core';
+import { css, SerializedStyles } from '@emotion/core';
 import { body } from '@guardian/src-foundations/typography';
 import { remSpace } from '@guardian/src-foundations';
+import { textSans } from 'styles';
+import { Format, Design } from '@guardian/types/Format';
 
 // ----- Component ----- //
 
 interface Props {
     children?: ReactNode;
+    format: Format;
 }
 
-const styles = css`
-    ${body.medium()}
-    overflow-wrap: break-word;
-    margin: 0 0 ${remSpace[3]};
-`;
+const styles = (design: Design): SerializedStyles => {
+    const advertisementFeature = design === Design.AdvertisementFeature
+        ? textSans
```

Comment:
Would it be preferable to use a design system font here? E.g. `textSans.medium()`?

```ts
const font = design === Design.AdvertisementFeature ? textSans.medium() : body.medium();

return css`
    ${font}
    overflow-wrap: break-word;
    margin: 0 0 ${remSpace[3]};
`;
```

Reply:
`textSans.medium()` is a lot smaller than `headline.medium()`. Even the largest textSans is smaller that `headline.medium()`